context/.start_jupyter_run_in_rapids.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
-/rapids/utils/start_jupyter.sh > /dev/null
+
+OUTPUT="/dev/null"
+
+# Run Jupyter in foreground if $JUPYTER_FG is set
+if [[ -v JUPYTER_FG ]]; then
```

Comment:
Any reason specifically to check if its set versus set to something like `true`? I just don't want someone to do `JUPYTER_FG=false` when that would still trigger the behavior.

Reply:
good call, I'll push an update to specifically check for `true`.

Reply:
updated:

![image](https://user-images.githubusercontent.com/7400326/89690021-9522b400-d8d3-11ea-8b42-4f0ea180cef6.png)